tests/webview/integration/test_rex_redirects.py

```diff
@@ -28,6 +28,8 @@ def test_redirect_for_rex_books(webview_base_url, rex_base_url):
     assert rex_base_url in response.history[0].headers["location"]
 
 
+@markers.test_case("C553086")
+@markers.slow
```

Comment:
Soo, you have to put slow on all of these for them to be updated in testrail?

Reply:
yes. otherwise no testrail marking

Reply:
@m1yag1 removed some of the slow markers as per our discussion and pushed changes

Reply:
both test fails are the same as previously explained (removed app link -> PR waiting to be approved and redirected book). not related.